docs/typescript.md

```diff
@@ -253,6 +253,55 @@ const { order } = await solo.api.placeOrder({
 });
 ```
 
+#### Place Order(v2)
+```javascript
+import { ApiSide, ApiMarketName, BigNumber } from '@dydxprotocol/solo';
+
+// order has type ApiOrder
+const { order } = await solo.api.placeCanonicalOrder({
+  order: {
+    //Side your order is being placed on
+    side: ApiSide.BUY
+
+    //Market the trade is in
+    market: ApiMarketName.WETH_DAI
+
+    // denominated in base units. i.e. 1 ETH = 1e18
+    amount: new BigNumber('1e18'),
+
+    // denominated in base/quote.
+    price: '2.1',
```

Comment:
make the price closer to actual price of ETH, also explain why you would want 'e-12' for USDC markets